src/guc.c

```diff
@@ -789,6 +790,17 @@ _guc_init(void)
 							 NULL,
 							 NULL);
 
+	DefineCustomBoolVariable(MAKE_EXTOPTION("enable_uuid_compression"),
+							 "Enable uuid compression functionality",
+							 "Enable uuid compression",
+							 &ts_guc_enable_uuid_compression,
+							 false,
```

Comment:
should we have the default be true here, its gonna be improvement over current, not sure it makes sense to wait 1 minor release